hardpy/hardpy_panel/api.py

```diff
@@ -361,6 +361,57 @@ def set_manual_collect_mode(mode_data: dict) -> dict:
     return {"status": "success", "manual_collect_mode": enabled}
 
 
+@app.get("/api/storage_type")
+def get_storage_type() -> dict:
+    """Get the configured storage type.
+
+    Returns:
+        dict[str, str]: storage type ("json" or "couchdb")
+    """
+    config_manager = ConfigManager()
+    return {"storage_type": config_manager.config.database.storage_type}
+
+
+@app.get("/api/json_data")
+def get_json_data() -> dict:
+    """Get test run data from JSON storage.
+
+    Returns:
+        dict: Test run data from JSON files
+    """
+    config_manager = ConfigManager()
+    storage_type = config_manager.config.database.storage_type
+
+    if storage_type != StorageType.JSON:
+        return {"error": "JSON storage not configured"}
+
+    try:
+        storage_dir = Path(config_manager.config.database.storage_path) / "storage"
+        runstore_file = storage_dir / "runstore.json"
+
+        if not runstore_file.exists():
+            return {"rows": [], "total_rows": 0}
+
+        with runstore_file.open("r") as f:
+            data = json.load(f)
```

Comment:
1. In the current implementation, the same data is written to the **statestore** document as to the **runstore** for JSON. However, these are different databases for CouchDB. You can read more about this in the documentation:
https://everypinio.github.io/hardpy/about/frontend_sync/#frontend-data-synchronization

The **runstore** database is similar to **statestore** database, but there are differences:

- **runstore** contains the **artifact** field for test run, module, and case.
- **runstore** does not contain some fields: **progress**, **dialog_box**, **attempt**,
  **alert**, **operator_data**, **operator_msg**.

2. This is most likely why the functionality of dialogue boxes and operator messages does not work. Relevant examples (run in json mode):

* https://github.com/everypinio/hardpy/tree/main/examples/dialog_box
* https://github.com/everypinio/hardpy/tree/main/examples/operator_msg

```suggestion
        statestore_file = storage_dir / "statestore.json"

        if not statestore_file.exists():
            return {"rows": [], "total_rows": 0}

        with statestore_file.open("r") as f:
            data = json.load(f)
```